src/Symfony/Component/Form/Attribute/Type.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Attribute;
+
+/**
+ * Add an AsFormType class property as a FormType's field.
+ *
+ * @author Benjamin Georgeault <git@wedgesama.fr>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY)]
+final readonly class Type
```

Comment:
To stay consistent with the purpose of this attribute, as described in the class description, it represents a form field, so I’d prefer to name this class `Field` ?

Reply:
I think `Field` is wrong. Because it's biased by how it displays. But a "Field" could be an entire form. so I think we should keep `Type` but maybe rename other occurences of Field's to Type's. WDYT ?